ssl/record/ssl3_record_tls13.c

```diff
@@ -52,8 +53,18 @@ int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int send)
         rec->input = rec->data;
         return 1;
     }
+
     ivlen = EVP_CIPHER_CTX_iv_length(ctx);
 
+    /*
+     * To get here we must have selected a ciphersuite - otherwise ctx would
+     * be NULL
+     */
+    assert(s->s3->tmp.new_cipher != NULL);
+    if (s->s3->tmp.new_cipher == NULL)
+        return -1;
+    alg_enc = s->s3->tmp.new_cipher->algorithm_enc;
```

Comment:
I don't understand where `s->s3->tmp.new_cipher` would change between the start of this function and here, can you tell me?